src/Apps/Order/Routes/Respond/index.tsx

```diff
@@ -212,141 +209,128 @@ export class RespondRoute extends Component<RespondProps, RespondState> {
     })
   }
 
-  render() {
-    const { order, isCommittingMutation } = this.props
-
-    const artworkId = order.lineItems?.edges?.[0]?.node?.artwork?.slug
-
-    return (
-      <OrderRouteContainer
-        currentStep="Respond"
-        steps={counterofferFlowSteps}
-        content={
-          <Flex
-            flexDirection="column"
-            style={isCommittingMutation ? { pointerEvents: "none" } : {}}
+  return (
+    <OrderRouteContainer
+      currentStep="Respond"
+      steps={counterofferFlowSteps}
+      content={
+        <Flex
+          flexDirection="column"
+          style={isCommittingMutation ? { pointerEvents: "none" } : {}}
+        >
+          <Flex flexDirection="column">
+            <CountdownTimer
+              action="Respond"
+              note="Expiration will end negotiations on this offer. Keep in mind the work can be sold to another buyer in the meantime."
+              countdownStart={order.lastOffer?.createdAt!}
+              countdownEnd={order.stateExpiresAt!}
+            />
+            <OfferHistoryItem order={order} />
+            <TransactionDetailsSummaryItem
+              order={order}
+              useLastSubmittedOffer
+            />
+          </Flex>
+          <Spacer mb={[2, 4]} />
+          <RadioGroup
+            onSelect={responseOption =>
+              setResponseOption(responseOption as ResponseOptions)
```

Comment:
**thought (non-blocking):** I wish we could guarantee that the `value` attribute of each radio button comes from one of the `ResponseOptions`.

Reply:
Yeah, I was just thinking that it's a shame to have to typecast `responseOption`. I didn't realize that `BorderedRadio` was a library component. Thanks!